api/infrastructure/parallelcluster-api.yaml

```diff
@@ -309,719 +320,6 @@ Resources:
-          - Action:
-              - lambda:CreateFunction
-              - lambda:TagResource
-              - lambda:DeleteFunction
-              - lambda:GetFunctionConfiguration
-              - lambda:GetFunction
-              - lambda:InvokeFunction
-              - lambda:AddPermission
-              - lambda:RemovePermission
-              - lambda:UpdateFunctionConfiguration
-              - lambda:ListTags
-              - lambda:UntagResource
-            Resource:
-              - !Sub arn:${AWS::Partition}:lambda:${Region}:${AWS::AccountId}:function:parallelcluster-*
-              - !Sub arn:${AWS::Partition}:lambda:${Region}:${AWS::AccountId}:function:pcluster-*
-            Effect: Allow
-            Sid: Lambda
-          - Action:
-              - s3:*
-            Resource:
-              - !Sub arn:${AWS::Partition}:s3:::parallelcluster-*
-              - !Sub arn:${AWS::Partition}:s3:::aws-parallelcluster-*
-            Effect: Allow
-            Condition: !If
-              - IsMultiRegion
-              - !Ref AWS::NoValue
-              - StringEquals:
-                  aws:RequestedRegion:
-                    - !Ref Region
-            Sid: S3ResourcesBucket
-          - Action:
-              - s3:Get*
-              - s3:List*
-            Resource: !Sub arn:${AWS::Partition}:s3:::${Region}-aws-parallelcluster*
-            Effect: Allow
-            Condition: !If
-              - IsMultiRegion
-              - !Ref AWS::NoValue
-              - StringEquals:
-                  aws:RequestedRegion:
-                    - !Ref Region
-            Sid: S3ParallelClusterReadOnly
-          - Action:
-              - fsx:*
-            Resource:
-              - !Sub arn:${AWS::Partition}:fsx:${Region}:${AWS::AccountId}:*
-            Effect: Allow
-            Sid: FSx
-          - Action:
-              - elasticfilesystem:*
-            Resource:
-              - !Sub arn:${AWS::Partition}:elasticfilesystem:${Region}:${AWS::AccountId}:*
-            Effect: Allow
-            Sid: EFS
-          - Action:
-              - logs:DeleteLogGroup
-              - logs:PutRetentionPolicy
-              - logs:DescribeLogGroups
-              - logs:CreateLogGroup
-              - logs:TagResource
-              - logs:UntagResource
-              - logs:DescribeMetricFilters
-              - logs:PutMetricFilter
-              - logs:deleteMetricFilter
-            Resource: '*'
-            Effect: Allow
-            Condition: !If
-              - IsMultiRegion
-              - !Ref AWS::NoValue
-              - StringEquals:
-                  aws:RequestedRegion:
-                    - !Ref Region
-            Sid: CloudWatchLogs
-          - Action:
-              - resource-groups:ListGroupResources
-              - resource-groups:GetGroupConfiguration
-            Resource: '*'
-            Effect: Allow
-            Sid: ResourceGroupRead
-
-
-  ### IMAGE ACTIONS POLICIES
```

Comment:
question, we should be able to remove all the image policies from the parallelcluster policies file too, right?

Reply:
I believe many are also needed for the `build-image` operation. It might be worth a second pass (separately) to make sure they are scoped down as well now that we're not using this pathway for creating the API.